helm/charts/keycloak/keycloak-resources/restore-keycloak-settings.sh

```
@@ -0,0 +1,2 @@
+echo 'Restoring keycloak settings...'
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)